### DISCUSSION TITLE
How to set page title?

### DISCUSSION THREAD

**Discussion by iiabc** [fuma-nama/fumadocs#951] [Q&A] [answered]
I only know how to set the docs title, but how to set the titles of other pages (such as home)?

**Comment by Gitstar-OC** [comment]
Where do you want to set it ?

**Reply by Gitstar-OC** [reply]
Please explain more with a screenshot or text.

**Reply by iiabc** [reply]
![image](https://github.com/user-attachments/assets/db568140-c040-4797-8f93-ca9b64b8624b)

![image](https://github.com/user-attachments/assets/dc7b5f9b-6331-491d-a2b5-122bbd58031b)

I want to set the title of the web page label (the text to the right of the favicon)

**Reply by Gitstar-OC** [reply, marked as answer]
oh, you can set it here 
![image](https://github.com/user-attachments/assets/b6dc2cc2-130b-4795-9d6c-9562e5735eb9)

In the layout.tsx file or you can use the [next-seo](https://github.com/garmeeh/next-seo) for it.

**Reply by fuma-nama** [reply]
No you should use Next.js Metadata API for that, you might have to learn Next.js App Router basics from its docs, they have a detailed guide

**Reply by iiabc** [reply]
> No you should use Next.js Metadata API for that, you might have to learn Next.js App Router basics from its docs, they have a detailed guide

I don't have time to learn that much, I want it to work out of the box

**Reply by fuma-nama** [reply]
I honestly don't think with the help of docs and AI, it would be time consuming to learn the metadata API. It is straightforward in all Next.js APIs (especially compared to caching). 
https://nextjs.org/docs/app/building-your-application/optimizing/metadata

**Comment by Gitstar-OC** [comment]
Refer to this https://fumadocs-dev.vercel.app/docs/ui/page-conventions

**Comment by iiabc** [comment]
Okay, I understand. Edit in (home)/layout.tsx instead of page.tsx, and add the following code:
```tsx
import type { Metadata } from 'next'

export const metadata: Metadata = {
  title: '...',
}
```

**Reply by Gitstar-OC** [reply]
yes, you can also add description and export it or use generateMetaData.
